src/app/prevalence/components/SelectionContext.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { createContext, useContext, useState, ReactNode } from "react";
+
+// Define the shape of the context data
+interface SelectionContextData {
+    selectedMicroorganisms: string[];
+    setSelectedMicroorganisms: (microorganisms: string[]) => void;
+    selectedAnimalSpecies: string[];
+    setSelectedAnimalSpecies: (species: string[]) => void;
+}
+
+// Create the context with an initial undefined value
+const SelectionContext = createContext<SelectionContextData | undefined>(
+    undefined
+);
+
+// Create a custom hook to use the context
+export const useSelection = (): SelectionContextData => {
```

Comment:
PLease rename the hook to: usePrevalenceFilters, or something more specific - useSelection is too general